contracts/Staking.sol

```diff
@@ -439,6 +439,11 @@ contract Staking is AccessControl, StableThenToken, RewardCalculator, IERC1363Re
         disabled = true;
     }
 
+    /// Allows an admin to change the token price during the stable phase
+    function setTokenPrice(uint256 _tokenPrice) public onlyRole(DEFAULT_ADMIN_ROLE) stablePhaseOnly {
+        tokenPrice = _tokenPrice;
+    }
+
```

Comment:
in line with my other question, if you're upgrading the contract, you should probably keep the v1 version in the codebase

Reply:
yes, that would be an approach. as for the directory structure, it's whatever you prefer. I don't think there's a convention there